benchmarks/benchmarks/bench_core.py

```diff
@@ -276,10 +276,10 @@ def time_sum(self, dtype, size):
 
 class NumPyChar(Benchmark):
     def setup(self):
-        self.A = np.array([100*'x', 100*'y'])
+        self.A = np.array([100 * 'x', 100 * 'y'])
```

Comment:
Is it possible to avoid spacing around `*`? It isn't bad here, but IIRC, a topic that has been raised before about Black.

Reply:
I don't think ruff (or pycodestyle for that matter) can make a specific exception for `*`.